include/config/config_movement.h

```diff
@@ -68,14 +68,11 @@
 // Allows Mario to grab hangable ceilings from any state.
 #define HANGING_FIX
 
-// The last frame after hitting a wall that will be considered a firsty when wallkicking.
-#define FIRSTY_LAST_FRAME 1
-
 // The maximum angle the player can wall kick, in degrees. 0..90. To allow 45 degree wall kicks, you must supply `46` to allow 45 and under.
 #define WALL_KICK_DEGREES 45
 
-// This is vanilla behavior, disable it to allow ledge grabbing regardless of floor pitch.
-// #define LEDGE_GRABS_CHECK_SLOPE_ANGLE
+// Makes Mario unable to ledge grab steep slopes to prevent false ledge grabs.
+#define DONT_LEDGE_GRAB_STEEP_SLOPES
```

Comment:
this should be positively worded ideally, I don't have a suggestion atm

Reply:
`LEDGE_GRAB_STEEP_SLOPES`? :)